diagnostic.md

```diff
@@ -13,29 +13,25 @@ Record your responses inside the fenced code blocks below each question.
     `campus`?
```

Comment:
Question 1
I would take a chance and try to answer even if you're unsure. A wrong answer allows us to figure out how you're thinking about things and respond to that.

0/1